php/libraries/Database.class.inc

```diff
@@ -302,17 +337,30 @@ class Database
      *
      * Inserts a single row into the specified table, containing the values specified
      *
-     * @param string $table      the table into which to insert the row
-     * @param array  $set        the values with which to fill the new row
-     * @param bool   $autoescape determines whether the values to be set should
-     *                           automatically have the html escaped
-     * @param bool   $ignore     determines whether the insert throws and error or
-     *                           is discarded when value exists in DB
+     * @param string $table             the table into which to insert the row
+     * @param array  $set               the values with which to fill the new row
+     * @param bool   $autoescape        determines whether the values to be set
+     *                                  should automatically have the html escaped
+     * @param bool   $ignore            determines whether the insert throws and
```

Comment:
small typo: `determines whether the insert throws and error` -> `determines whether the insert throws an error`.